api/external/applications/applications.proto

```diff
@@ -138,66 +249,100 @@ service ApplicationsService {
   };
 }
 
+// Request message for GetServices.
 message ServicesReq {
+  // Search bar filters to apply to services, in the format of fieldname:value.
   repeated string filter = 1;
+  // Pagination parameters to apply to the returns services list.
   common.query.Pagination pagination = 2;
+  // Sorting parameters to apply to the returns services list.
   common.query.Sorting sorting = 3;
 }
 
+// Request message for GetServicesDistinctValues.
 message ServicesDistinctValuesReq {
+  // Field name of service values to retrieve.
   string field_name = 1;
+  // Text of value to query, supports wildcards (* and ?).
   string query_fragment = 2;
+  // Filters to apply to values, in the format of fieldname:value.
   repeated string filter = 3;
 }
 
+// Request message for GetServicesBySG.
 message ServicesBySGReq {
+  // Service group ID of the services.
   string service_group_id = 1;
+  // Pagination parameters to apply to the returned list of services.
   common.query.Pagination pagination = 2;
+  // Sorting paramters to apply to the returned list of services.
```

Comment:
nit: parameters

Reply:
This makes me curious how much work it'd be to hook up our spellchecker to the API docs. 🤔